### PR TITLE
[cross_file] fixed lastModified unimplemented error description

### PR DESCRIPTION
**Pull request by balvinderz**
Fixed lastModified unimplemented error description

Fixes https://github.com/flutter/flutter/issues/103930

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [X] I signed the [CLA].
- [X] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [X] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by balvinderz** [comment]
@ditman Can you please review this 😄

**Comment by Hixie** [comment]
test-exempt: error message contents

technically you could test this, call each of the unimplemented methods and see if they throw the right message. not sure it's worth it in practice...

**Review by ditman — APPROVED** [review]
LGTM!

**Comment by ditman** [comment]
> technically you could test this, call each of the unimplemented methods and see if they throw the right message. not sure it's worth it in practice...

@Hixie I was going to say that we've never had any report from having a bad text on an UnimplementedException but... we actually just did! This problem hasn't been a significant source of issues until this one popped up, though.

**Comment by ditman** [comment]
We need a second reviewer on this one, ping @stuartmorgan?

File: packages/cross_file/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.3.3+1
+
+* Fixed lastModified unimplemented error description.
```
Comment (stuartmorgan) [inline packages/cross_file/CHANGELOG.md:3]: Fixes

See the changelog style guide linked from the PR checklist.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/cross_file/CHANGELOG.md (hunk 1 above)
Comment (stuartmorgan) [reply packages/cross_file/CHANGELOG.md:3]: Also, please add backticks around `lastModified` since it's code.

Comment (balvinderz) [reply packages/cross_file/CHANGELOG.md]: fixed

**Review by stuartmorgan — APPROVED** [review]

**Pull request merged**